src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx

```diff
@@ -1099,35 +1102,36 @@ To set permanent data storage on Docker:
 
 1. Create a directory on the host where you are launching the Job Manager. This is your source directory.
 2. Launch the Job Manager, mounting the source directory to the target directory `/var/lib/newrelic/synthetics`.
-		
 
-Example:
 
-```
-		docker run ... -v /sjm-volume:/var/lib/newrelic/synthetics:rw ...
-```
+    Example:
+
+    ```sh
+    docker run ... -v /sjm-volume:/var/lib/newrelic/synthetics:rw ...
+    ```
```

Comment:
```suggestion
Example:

```sh
  docker run ... -v /sjm-volume:/var/lib/newrelic/synthetics:rw ...
```
```

Reply:
Granted, I have not previewed the change in VS Code (I still need to learn how to do that), but the goal here is to nest the Example under the itemized list. This becomes especially obvious when not nesting breaks the counter of the itemized list. Example:
![image](https://github.com/user-attachments/assets/82bfbe70-6cd2-4a32-b097-6d15f8ca7595)